eventually/authentication/models.py

```diff
@@ -34,7 +35,22 @@ class CustomUser(AbstractBaseUser):
 
     def __str__(self):
         """
-            This magic method is redefined to show first, last name and id of the CustomUser object.
+        This magic method is redefined to show first, last name and id of the CustomUser object.
+        :return: Full string representation of the object
+        """
+        return '{} {} {} {} {} {} {} {}'.format(self.id,
+                                                self.first_name,
+                                                self.middle_name,
+                                                self.last_name,
+                                                self.email,
+                                                self.password,
+                                                self.updated_at,
+                                                self.created_at)
+
+    def __repr__(self):
+        """
+        This magic method is redefined to show first, last name and id of the CustomUser object.
+        :return: Short string representation of the object
```

Comment:
indicate which certain fields are returned by `return` statement of this method.

Reply:
Fixed.